style.css

```diff
@@ -262,180 +308,152 @@ input[type=submit]:hover{
 	clear: both;
 }
 
-/*--------Home Content Box------------*/
-
```

Comment:
Removed content_2 from this part of the CSS style sheet as I have put it in a more suitable location